configure-c8y.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+
+if [ `id -u` != 0 ] ; 
```

Comment:
Please use the newer syntax `$(id -u)` and if you are going to use `;` then include `then` on the same line.

But generally please correct all of the [shellcheck](https://www.shellcheck.net/) errors. There are generally plugins in most IDE that can validate your shell script (e.g. VSCode has `timonwong.shellcheck`)

```suggestion
if [ "$(id -u)" != 0 ]; then
    printf "Please run as sudo or root!\n"
    exit
fi
```